TestProjects/ShaderGraph/Assets/CommonAssets/Editor/ShaderGraphUITestHelpers.cs

```diff
@@ -0,0 +1,124 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using UnityEditor;
+using UnityEditor.ShaderGraph.UnitTests;
+using UnityEngine;
+using UnityEngine.UIElements;
+using UnityEngine.UIElements.StyleSheets;
+using UIECursor = UnityEngine.UIElements.Cursor;
+
+namespace UnityEditor.ShaderGraph.UnitTests
+{
+    static class ShaderGraphUITestHelpers
+    {
+        private static EventBase CreateEvent(Event evt)
+        {
+            Assembly uiElementAssembly = null;
+
+            foreach (var assembly in AppDomain.CurrentDomain.GetAssemblies())
+            {
+                var assemblyName = assembly.GetName().ToString();
+                if (assemblyName.Contains("UnityEngine.UIElementsModule"))
+                {
+                    uiElementAssembly = assembly;
+                }
+            }
+            Type uiElementsUtilityType = uiElementAssembly?.GetType("UnityEngine.UIElements.UIElementsUtility");
```

Comment:
Thanks for reminding me of TypeCache, forgot that it was the preferred way, but in this particular case I don't think that will work because the type being derived from is an internal type, `IUIElementsUtility`, as is the derived type I'm actually referencing `UIElementsUtility`. Throws a compile error about the internal class access. :(